### PR TITLE
This cases is used for aide check

### PR DESCRIPTION
**Pull request by Jiawei517**
This case is focus on the basic function of aide. Compare the differences between the aide database and the file system.

### CODE REVIEW

File: tests/console/aide_check.pm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (coolo) [inline tests/console/aide_check.pm]: Just use 2016 for new tests

File: tests/console/aide_check.pm [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# test for basic function of aide. Check different between aide.db and file system
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+    sleep 2;
+    validate_script_output "zypper -n in aide", sub { m/(Installing.*)|(.*already installed)/ };
+    sleep 2;
+    assert_script_run "cp /etc/aide.conf /etc/aide.conf.bak";
+    sleep 2;
```
Comment (coolo) [inline tests/console/aide_check.pm]: why would you sleep 2 after an asserted cp? just remove all sleep 2 - and please tell me where you copy&pasted it from, so can we fix that test too

**Comment by coolo** [comment]
add it to extratests in main.pm of sle and opensuse?

**Comment by Jiawei517** [comment]
Haven't  added it to main.pm, I will update main.pm later after we define the teat plan of FIPS.

**Comment by coolo** [comment]
LGTM, but I would like to delay merging until it's sorted out where to use this test. I don't like unused tests - we have too many ;(

File: tests/console/aide_check.pm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    validate_script_output "zypper -n in aide", sub { m/(Installing.*)|(.*already installed)/ };
+    assert_script_run "cp /etc/aide.conf /etc/aide.conf.bak";
+    assert_script_run "sed -i \'s/^\\//!\\//\' /etc/aide.conf && sed -i \'s/^!\\/var\\/log/\\/var\\/log/\' /etc/aide.conf";
+    assert_script_run "aide -i";
+    assert_script_run "cp /var/lib/aide/aide.db.new /var/lib/aide/aide.db";
+    assert_script_run "aide --check";
+    assert_script_run "touch /var/log/testlog";
+    assert_script_run "clear";
+    script_run "aide --check";
+    assert_screen "aide_result", 10;
```
Comment (okurz) [inline tests/console/aide_check.pm]: can you use default timeout here? We try to use default timeout as in some cases really slow workers have problems and we can not rely on the SUT performance too much.

**Comment by sysrich** [comment]
Ping @Jiawei517 please can you answer @okurz's question?

And any news about the main.pm update?

File: tests/console/aide_check.pm (hunk 3 above)
Comment (Jiawei517) [inline tests/console/aide_check.pm]: I have updated it and set it as default in my local file.  I will commit it later after added it to main.pm.

**Comment by mitiao** [comment]
@sysrich @coolo , @QingmingSu will update fips in main.pm, unfortunately he is on 2 or 3 days sick leave.

**Comment by Jiawei517** [comment]
Updated main.pm,  added test entrance of this case.

**Comment by Jiawei517** [comment]
Hi @coolo @okurz, I have update main.pm and added enterance of the aide test case. Could you please help review it again.  Thanks :)

**Comment by okurz** [comment]
The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.
Rest looks ok. If others want to merge, go ahead.

**Comment by Jiawei517** [comment]
@okurz thanks for your comments, I have updated the commit.
Closed this PR by accident operation, and submit a new PR #1426 instead.

**Comment by okurz** [comment]
please reopen the PR and update it by pushing to the same branch. There should be no problem with reopening a PR.

**Pull request closed without merging**